modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,21 +17,30 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
-  // If they're in the same paragraph, get them to stack.
-  [class*="su-button"] {
-    display: block;
-    width: max-content;
-  }
+  // Allow the power user to stack buttons with .su-stack-button class.
+  .su-stack-button {
+    [class*="su-button"] {
+      display: block;
+      width: max-content;
+    }
 
-  // We want don't want to target the first link.
-  [class*="su-button"] + [class*="su-button"] {
-    @include margin(-0.9rem null null null);
-  }
+    // We want don't want to target the margin on the first button.
+    [class*="su-button"] + [class*="su-button"] {
+      @include margin(-1rem null null null);
+    }
+
+    &.text-align-center {
+      a {
+        margin-left: auto;
+        margin-right: auto;
+      }
+    }
 
-  // Allow the power user to get buttons to display inline.
-  .su-inline-button [class*="su-button"],
-  .su-inline-button [class*="su-button"] + [class*="su-button"] {
-    display: inline-block;
-    @include margin(null 0.5rem null null);
```

Comment:
Do we know if anyone is using these? Will we need to provide a release note about this change? Without doing some sort of scan of the sites, has this been documented anywhere?

Reply:
@cjwest ^

Reply:
:(

I'll unpublish this:
https://userguide.sites.stanford.edu/tour/paragraphs/text-area/working-with-buttons

Reply:
@sherakama,
Here's the ticket to create an alert:
https://stanfordits.atlassian.net/browse/D8CORE-2216

and to be complete, the ticket to update documentation:
https://stanfordits.atlassian.net/browse/D8CORE-2217